### PR TITLE
Feat: Suggest CW20 Token

### PR DESCRIPTION
**Pull request by DavideSegullo**
Hello, as described in issue #177 ,I am working on an implementation for CW20 token suggestion, if you have any feedback or suggestions I am open to any discussion, thanks!

### CODE REVIEW

File: wallets/cosmostation-extension/src/extension/client.ts [hunk 1]
```diff
@@ -32,6 +33,16 @@ export class CosmostationClient implements WalletClient {
     return this.client.providers.keplr;
   }
 
+  async suggestToken({ chainName, tokens }: SuggestToken) {
```
Comment (pyramation) [inline wallets/cosmostation-extension/src/extension/client.ts:36]: can we rename this to `suggestCW20Token`, or, `suggestToken` should take a parameter that is an enum or something, like `type: 'cw20' | 'erc20'`, etc.

**Review by pyramation — CHANGES_REQUESTED** [review]

**Comment by pyramation** [comment]
awesome work @DavideSegullo !!! thinking to make this more flexible in the future, we should either rename the function, or add a parameter. Perhaps the parameter option is more flexible/generic.

File: wallets/cosmostation-extension/src/extension/client.ts (hunk 1 above)
Comment (DavideSegullo) [reply wallets/cosmostation-extension/src/extension/client.ts:36]: Yes, that's a good idea, I'll add another parameter

**Comment by DavideSegullo** [comment]
> awesome work @DavideSegullo !!! thinking to make this more flexible in the future, we should either rename the function, or add a parameter. Perhaps the parameter option is more flexible/generic.

Yes, I'll also add some examples

**Comment by DavideSegullo** [comment]
@pyramation It seems fine, I also added an example

**Pull request merged**